src/More/index.tsx

```diff
@@ -1,204 +1,90 @@
 import {
-  ViewStyle,
-  View,
-  StyleSheet,
-  ScrollView,
-  TouchableHighlight,
-  TouchableOpacity,
-} from "react-native"
+  createStackNavigator,
+  StackNavigationOptions,
+} from "@react-navigation/stack"
 import { useTranslation } from "react-i18next"
-import { SvgXml } from "react-native-svg"
-import {
-  NavigationParams,
-  NavigationScreenProp,
-  NavigationState,
-} from "react-navigation"
 
-import { getLocalNames } from "../locales/languages"
-import { RTLEnabledText } from "../components/RTLEnabledText"
-import { NavigationBarWrapper } from "../components/NavigationBarWrapper"
-import { Stacks, Screens, useStatusBarEffect } from "../navigation"
+import MoreMenuScreen from "./Menu"
+import AboutScreen from "./About"
+import LicensesScreen from "./Licenses"
+import ENDebugMenu from "./ENDebugMenu"
+import ENLocalDiagnosisKeyScreen from "./ENLocalDiagnosisKeyScreen"
+import ExposureListDebugScreen from "./ExposureListDebugScreen"
+import LanguageSelection from "./LanguageSelection"
 
-import { Icons } from "../assets"
-import { Buttons, Colors, Spacing, Typography } from "../styles"
+import { Colors } from "../styles"
 
-interface SettingsScreenProps {
-  navigation: NavigationScreenProp<NavigationState, NavigationParams>
-}
+import { Screens } from "../navigation/index"
 
-interface LanguageSelectionListItemProps {
-  icon: string
-  iconLabel: string
-  label: string
-  onPress: () => void
+const Stack = createStackNavigator()
+
+const SCREEN_OPTIONS: StackNavigationOptions = {
+  headerStyle: {
+    backgroundColor: Colors.primaryViolet,
+  },
+  headerTitleStyle: {
+    color: Colors.white,
+    textTransform: "uppercase",
+  },
+  headerBackTitleVisible: false,
+  headerTintColor: Colors.white,
 }
-const LanguageSelectionListItem = ({
-  icon,
-  iconLabel,
-  label,
-  onPress,
-}: LanguageSelectionListItemProps) => (
-  <TouchableHighlight
-    underlayColor={Colors.underlayPrimaryBackground}
-    style={styles.listItem}
-    onPress={onPress}
-  >
-    <View style={{ flexDirection: "row", alignItems: "center" }}>
-      <SvgXml
-        xml={icon}
-        accessible
-        accessibilityLabel={iconLabel}
-        style={[styles.icon, { marginRight: Spacing.small }]}
-      />
-      <RTLEnabledText style={{ ...Typography.mainContent }}>
-        {label}
-      </RTLEnabledText>
-    </View>
-  </TouchableHighlight>
-)
 
-const SettingsScreen = ({ navigation }: SettingsScreenProps): JSX.Element => {
-  const {
-    t,
-    i18n: { language: localeCode },
-  } = useTranslation()
-  const languageName = getLocalNames()[localeCode]
-  useStatusBarEffect("light-content")
+type MoreStackRouteName =
+  | "Settings"
+  | "About"
+  | "Licenses"
+  | "ENDebugMenu"
+  | "LanguageSelection"
+  | "AffectedUserFlow"
+  | "ExposureListDebugScreen"
+  | "ENLocalDiagnosisKey"
 
-  const navigateTo = (screen: string) => {
-    return () => navigation.navigate(screen)
-  }
+interface MoreStackRouteState {
```

Comment:
for context, this typing would ideally be removed and replaced with something directly from the library, but I wasnt able to get it to line up from what is provided from their declarations so we needed to recreate it.

Reply:
Got it makes sense!

Reply:
We actually don't need this any more right? Given that we're not doing the tab bar visibility thing.

Reply:
> > I dig the refactor! what are your reasons for moving the react-navigation stack from the navigation folder to the view layer folder? It looks good to me, but curious to your reasoning.
> 
> @johnschoeman Good question. I kept expecting the stack navigator for the more screens to be at `More/index.tsx` and was confused when it was actually the menu screen. Makes more sense to me for the stack to live at `index.tsx` and the menu screen to live at `More/Menu.tsx`.

@johnschoeman Do you think this should live in `navigation/MoreStack.tsx` instead?

Reply:
I could see it going both ways tbh.  on the one hand being consistent and having all the navigators in one spot is nice. On the other hand grouping things by feature rather than implementation detail usually leads to a code base more aligned with business goals.  So long as we are consistent, I dont think it ultimately matters too much.  That said, I do see most projects keep all their navigators colocated so that is somewhat more of the idiomatic approach.

totally up to you on what you think is best, I think you have the most context on the navigation solution at this point.